### DISCUSSION TITLE
react-native version

### DISCUSSION THREAD

**Discussion by lucasAzS** [jpudysz/react-native-unistyles#85] [Q&A] [answered]
which version of react-native is needed for using v2?

**Comment by jpudysz** [comment]
I can't say exactly, since my build tools are incompatible, but it should be at least 0.68, or possibly 0.66, when Meta first shipped the JSI bindings.

**Reply by lucasAzS** [reply]
I'm with an error similar to this, later will open a issue with details https://github.com/jpudysz/react-native-unistyles/issues/72

**Reply by jpudysz** [reply]
Make sure you followed all instructions: https://reactnativeunistyles.vercel.app/reference/errors/

**Comment by lucasAzS** [comment]
Ok I did revised all the setup steps and the instructions on [erros](https://reactnativeunistyles.vercel.app/reference/errors/) and on [faq](https://reactnativeunistyles.vercel.app/reference/faq/)

Even calling `import 'react-native-unistyles'` in my App.tsx the error `Error: Unistyles runtime is not available.` remains

The App is in an NX monorepo and is from my job, I dont know how to share an example without the risk to share someting confidential

**Reply by jpudysz** [reply, marked as answer]
Hey Lucas, I would like to help but I’m not sure how. This seems to be an issue specific to your setup, as no one else has reported the same problem.

Installing Unistyles isn’t complicated. Simply add it to the correct package.json and ensure you’ve imported something from the package. It should automatically display a log indicating that Unistyles has been installed.

If you can’t see such a message, consider adding a console.log to Unistyles’ index.ts in node_modules and figure out a way to call this file. The reason it’s not available is likely that the library hasn’t had a chance to load in your monorepo.

**Reply by lucasAzS** [reply]
Yeah, for some reason, the index.ts of the package is not being called. For now, we're going to remain in v1 and eventually revise the monorepo.
